baseapp_follows/README.md

```
@@ -1,85 +1,186 @@
 
-```
-git clone git@github.com:silverlogic/baseapp-backend.git
-```
+``python
+# In your settings.py or settings/base.py, add:
+BASEAPP_FOLLOWS_FOLLOW_MODEL = "follows.Follow"
+BASEAPP_FOLLOWS_FOLLOWABLEMETADATA_MODEL = "follows.FollowableMetadata"
+``
 
-And manually install the package:
+### Migrating from a legacy schema
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)